docs/logs/configuring.asciidoc

```diff
@@ -0,0 +1,105 @@
+[role="xpack"]
+[[xpack-logs-configuring]]
+
+:ecs-link: {ecs-ref}[Elastic Common Schema (ECS)]
+
+== Configuring the Logs UI
+
+The `filebeat-*` index pattern is used to query data by default. If your logs
+are located in a different set of indices, use a different timestamp field, or
+contain parsed fields which you want to expose as individual columns, you can
+adjust the source configuration via the user interface or the {kib}
+configuration file.
+
+NOTE: Logs and Infrastructure share a common data source definition in
+each space. Changes in one of them can influence the data displayed in the
+other.
```

Comment:
Maybe we should mention "This will be fixed in an upcoming release" ?

Reply:
I'm not sure if the official docs are the right place to talk about planned features. :thinking: